### PR TITLE
Dashboard Bugfix: Favourites Action in all Dashboard Blocks

### PR DESCRIPTION
**Pull request by fhelfer**
https://mantis.ilias.de/view.php?id=37869

addToFavourites() was not in action menu for Blocks in Tile View also entirely missing for Learning Sequences
Only trunk

### CODE REVIEW

**Comment by dsstrassner** [comment]
Is there already mantis issue?

**Comment by fhelfer** [comment]
there is not, but i can create one

**Comment by fhelfer** [comment]
https://mantis.ilias.de/view.php?id=37869

**Review by iszmais — CHANGES_REQUESTED** [review]
@fhelfer Thank you again for you continous effort.

This seems to work, but is missing a differentation when the item is already within the favourites (than an action to remove the item should be shown).
This is also still broken in the list view, therefore i would appreciate a fix for both views in one PR.

Thank you in advance!

File: Services/Dashboard/Block/classes/class.ilDashboardBlockGUI.php [hunk 1]
```diff
@@ -29,12 +28,13 @@
 
 /**
  * @ilCtrl_IsCalledBy ilDashboardBlockGUI: ilColumnGUI
- * @ilCtrl_Calls ilDashboardBlockGUI: ilCommonActionDispatcherGUI
+ * @ilCtrl_Calls      ilDashboardBlockGUI: ilCommonActionDispatcherGUI
```
Comment (lscharmer) [inline Services/Dashboard/Block/classes/class.ilDashboardBlockGUI.php]: Please remove the superfluous whitespace.

**Review by lscharmer — CHANGES_REQUESTED** [review]

**Review by lscharmer — APPROVED** [review]

**Review by iszmais — APPROVED** [review]

**Comment by iszmais** [comment]
@fhelfer Thank you again for the fast fix.

sincerely,
@iszmais

**Pull request merged**